airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectorMetadataDefinitionV0.yaml

```diff
@@ -53,6 +53,16 @@ properties:
         format: uri
       githubIssueLabel:
         type: string
+      maxSecondsBetweenMessages:
```

Comment:
Should we have a default value for this field?

Reply:
Since its an optional override Im thinking no

Reply:
nit: can we document the default value? Either in the description or as an actual default is fine

Reply:
So this is where we are a bit caught between our two repos.

Our `airbyte` repo holds our connectors

Then our `platform` repo hold the system that runs these connectors

This PR is in relation to the metadata service which lives in `airbyte` and creates the registry that the `platform` consumes.

What that means in practise is 
1. we have duplicated types/fields between the metadata service in `airbyte` and the platform code in `platform-internal`. (At least in the short term)
2. We have the config of a connector separate from the implementation logic that consumes that config 

Some additional context here: https://docs.google.com/document/d/1jl122NkgH9rs6mr58UgyOVqD9J4d_656BcCEMlVVfbQ/edit#

For this field the default is implemented [in the platform repo here](https://github.com/airbytehq/airbyte-platform-internal/blob/master/oss/airbyte-featureflag/src/main/kotlin/FlagDefinitions.kt#L56) which means the default used in platform can change, and this system in `airbyte` would be none the wiser. 

So we can list a default value in the description, but I think given enough time it will become incorrect